mobile-save-for-later/src/main/scala/com/gu/sfl/lib/SavedArticlesMerger.scala

```diff
@@ -24,6 +24,9 @@ class SavedArticlesMergerImpl(savedArticlesMergerConfig: SavedArticlesMergerConf
       case Success(Some(articles)) =>
         logger.debug(s"success persisting articles for ${userId}")
         Right(articles)
+      case Success(None) =>
```

Comment:
Compiler complained this case was missing; so I added it. Should check there are no side effects.